canonical/canonical.com
# WD-10729 - update contact form

## Done

Updated contact form according to copy doc

## QA

- [demo link](https://canonical-com-1270.demos.haus/contact-us)
- [copy doc](https://docs.google.com/document/d/1m26cLDsx5OoCBFSblGgEUiGHsRtdLWAI6GBh-KnRIq0/edit)
- View the site on demo
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the forms are correct according to copy doc

## Issue / Card
[WD-10729](https://warthogs.atlassian.net/browse/WD-10729)


[WD-10729]: https://warthogs.atlassian.net/browse/WD-10729?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
